DS_and_Algorithm/Day3/Selectionsort.py

```diff
@@ -0,0 +1,22 @@
+def selection_sort(arr):
+    n = len(arr)
+
+    for i in range(n):
+        # Find the minimum element in the unsorted part
+        min_index = i
+        for j in range(i-1, n):
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider fixing inner loop range</b></div>
<div id="fix">

The range in the inner loop `range(i-1, n)` appears incorrect for selection sort. It should be `range(i+1, n)` to compare with unsorted elements after the current position.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -7 +7 @@
 -        for j in range(i-1, n):
 +        for j in range(i+1, n):
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/Amruta101998/DS-and-Algorithm/pull/31#issuecomment-2535533488>#6662a4</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged